src/module/ClearCompleteTask.js

```diff
@@ -0,0 +1,14 @@
+import TodoClass from './todolistClass.js';
+
+// clear complete task button
+const ClearCompFunction = () => {
+  const ObjtodoClass = new TodoClass();
+  const getListData = ObjtodoClass.GetTodoList();
+  const UncompleteTask = getListData.filter((item) => item.completed === false);
+  UncompleteTask.forEach((item, index) => {
+    item.index = index + 1;
+  });
+  localStorage.setItem('todoListStorage', JSON.stringify(UncompleteTask));
+  window.location.reload();
```

Comment:
- [ ] [OPTIONAL] I am wondering why you have made your page reload after this method runs, I think you do not need it. So I suggest you remove it.